sourcemash/frontend/static/js/collections/items.js

```diff
@@ -1,12 +1,17 @@
 Sourcemash.Collections.Items = Backbone.Collection.extend({
     model: Sourcemash.Models.Item,
     url: function() {
-        return '/api/feeds/' + this.feed.get('id') + '/items'
+        if (this.feed) {
+            return '/api/feeds/' + this.feed.get('id') + '/items'
+        } else if (this.category) {
+            return '/api/categories/' + this.category.get('category') + '/items'
+        }
```

Comment:
We'd have to pass {feed: feed, category: category} for this to happen.

I'm trying to think of a time when a collection of items is filtered by both a feed and a category. What I came up with is this: if we want to allow users to filter a feed's items by category (or filter a category's items by feed), it would probably make more sense to have this dealt with as a filter function on the collection (see: http://stackoverflow.com/questions/11762105/filter-backbone-collection-by-attribute-value).

So I don't think we will have both this.feed and this.category. That being said, if both happened to be defined by some bug it would just resort to using the feed_id, which I think is satisfactory.